pkg/generate/app/cmd/describe.go

```diff
@@ -171,11 +171,18 @@ func describeBuildPipelineWithImage(out io.Writer, ref app.ComponentReference, p
 				fmt.Fprintf(out, "      * The resulting image will be pushed to %s %q\n", to.Kind, to.Name)
 			}
 		}
-		if len(trackedImage) > 0 {
-			if noSource {
-				fmt.Fprintf(out, "      * Use 'start-build --from-dir=DIR' to trigger a new build\n")
-			} else {
+
+		if noSource {
+			// if we have no source, the user must always provide the source from the local dir(binary build)
+			fmt.Fprintf(out, "      * Use 'start-build --from-dir=DIR' to trigger a new build\n")
```

Comment:
[ ] would be unclear imho since that usually means optional.  and angle brackets usually mean "substitute this with your value".  so i'm not sure what means "pick one of the 3 things here"